v3-nginx/nginx.conf

```diff
@@ -0,0 +1,33 @@
+load_module modules/ngx_http_modsecurity_module.so;
+
+user ${USER};
+worker_processes 1;
+
+error_log ${ERRORLOG} ${LOGLEVEL};
+pid /var/run/nginx.pid;
+
+events {
+    worker_connections ${WORKER_CONNECTIONS};
+}
+
+http {
+
+    modsecurity on;
+    modsecurity_rules_file /etc/modsecurity.d/setup.conf;

```

Comment:
These lines should probably go into `/etc/nginx/conf.d/modsecurity.conf`, or so, correct?